api/src/main/java/org/openmrs/module/locationbasedaccess/PatientSearchAdviser.java

```diff
@@ -52,37 +52,37 @@ public Advice getAdvice() {
 
     private class PatientSearchAdvise implements MethodInterceptor {
         public Object invoke(MethodInvocation invocation) throws Throwable {
-            Integer sessionLocationId = Context.getUserContext().getLocationId();
-            String locationAttributeUuid = Context.getAdministrationService().getGlobalProperty(LocationBasedAccessConstants.LOCATION_ATTRIBUTE_GLOBAL_PROPERTY_NAME);
             Object object = invocation.proceed();
```

Comment:
The actual problem occurred at this point, 
`Integer sessionLocationId = Context.getUserContext().getLocationId();`
daemon doesn't have any Location attribute. So that, I have cheked it before this line.

Reply:
Did you understand my comment?

Reply:
Yes got it @dkayiwa  and Fixed